russian/gusli/vodka/vodka.md

```diff
@@ -0,0 +1,5 @@
+Наконец-то вы добралсь до полторалитровой мутной самогонки, выгнанной судя по всему из зефира.
+Оторвав кусок зефира от стены, вы делаете несколько глотков прямо из сосуда и занюхиваете зефиром.
+"Что-то не то" - думаете вы. Быстрыми проворными движениями вы придаете зефиру форму шпротины и с жадностью заедаете выпитое.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.